src/java.desktop/macosx/classes/com/apple/laf/AquaInternalFrameBorder.java

```diff
@@ -380,8 +380,6 @@ public void drawWindowTitle(final Graphics g, final JInternalFrame frame, final
         final int w = inW;
         int h = inH;
 
-        h = metrics.titleBarHeight + inH;
```

Comment:
Just a suggestion, after the current change it is possible to align the h and other x/y/w vars and make it final.